test/services/homework/submissionService.test.js

```diff
@@ -4,19 +4,27 @@ const testObjects = require('../helpers/testObjects')(app);
 
 const { expect } = chai;
 
-
-async function createSubmission(teachers, students, publicSubmission = false, substitutionTeachers = [], courseStudents = [], createTeacherPrivateSubmission = false) {
+async function createSubmission(
+	teachers,
+	students,
+	publicSubmission = false,
+	substitutionTeachers = [],
+	courseStudents = [],
+	createTeacherPrivateSubmission = false
+) {
 	const originalTeacher = teachers[0];
 	const submitterId = students[0];
 	const studentIds = students.map((s) => s._id);
 	const courseStudentIds = courseStudents.map((s) => s._id);
 
-	const course = !createTeacherPrivateSubmission ? await testObjects.createTestCourse({
-		teacherIds: teachers.map((t) => t._id),
-		substitutionIds: substitutionTeachers.map((s) => s._id),
-		userIds: [...new Set([...studentIds, ...courseStudentIds])],
-		schoolId: originalTeacher.schoolId,
-	}) : undefined;
+	const course = !createTeacherPrivateSubmission
+		? await testObjects.createTestCourse({
+				teacherIds: teachers.map((t) => t._id),
```

Comment:
Codacy found an issue: [Expected indentation of 3 tabs but found 4.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)